libs/fetcher.client.js

```diff
@@ -41,7 +29,7 @@ function parseResponse(response) {
         try {
             return JSON.parse(response.responseText);
         } catch (e) {
-            debug('json parse failed:' + e, 'error', NAME);
+            debug('json parse failed:' + e, 'error');
```

Comment:
It was just the string `FetcherClient` but `debug` automatically annotates so it was unnecessary.